File: baekjoon/10818/BOJ10818_최소최대.js
let input = require('fs').readFileSync('/dev/stdin').toString().split('\n');

let n = Number(input[0]);
let arr = input[1].split(' ').map(x=> Number(x));

let max = arr[0]; 
let min = arr[0];

for (let i = 1; i < n; i++) {
  if (max < arr[i]) {
    max = arr[i];
  }
  if (min > arr[i]) {
    min = arr[i];
  }
}

console.log(`${min} ${max}`);

File: baekjoon/10818/BOJ10818_최소최대.py
n = int(input())

list = list(map(int, input().split()))

max = list[0]
min = list[0]

for i in range(1, n):
    if list[i] >= max:
        max = list[i]
    elif list[i] <= min:
        min = list[i]

print(min, max)